frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts

```diff
@@ -204,7 +204,7 @@ describe('Interacting with a `OwnNamespace` install mode Operator (Prometheus)',
     expect(crudView.successMessage.getText()).toContain('example has been updated to version');
   });
 
-  it('displays Kubernetes objects associated with the `Prometheus` in its "Resources" section', async () => {
+  xit('displays Kubernetes objects associated with the `Prometheus` in its "Resources" section', async () => {
```

Comment:
This test needs to be disabled?

Reply:
> This test needs to be disabled?

@benjaminapetersen I mean to push that to #5100, removed from this PR